### DISCUSSION TITLE
How do I configure this extension for kubernetes patches?

### DISCUSSION THREAD

**Discussion by vbezhenar** [redhat-developer/vscode-yaml#985] [Q&A]
I have files which belong to kubernetes deployments and files which patch those deployments. Obviously patches are not valid kubernetes deployments, they miss required fields.

Ideally I'd like to configure extension to recognize patches, however ignoring schema for patches is also fine for me.

My names are like `deploy.yaml`, `deploy-name.yaml`, `deploy-name-patch.yaml`. First two are valid kubernetes deployments, third is patch.

I tried to configure this extension in the following way:

```
  "yaml.schemas": {
    "none": ["*-patch.yaml"],
    "Kubernetes": ["deploy*.yaml"]
  }
```

This does not work for patches. It works correctly for deployments, but for patches both schemas are applied, schema `none` does not even exist, so it just does not work well.

**Comment by tysen** [comment]
In case anybody else finds this looking for a solution, [try this](https://github.com/vscode-kubernetes-tools/vscode-kubernetes-tools?tab=readme-ov-file#yaml-processing-directives).
